cypress/e2e/link.feature

```diff
@@ -0,0 +1,64 @@
+Feature: Test link
```

Comment:
`Test link creation`

Reply:
done and I rename file in `link-creation.feature`